proposals/evaluation/uc-3-inheritance.md

```diff
@@ -236,19 +240,29 @@ However, if WAC's use of `acl:default` were to be relaxed as described in [issue
 
 See also: https://github.com/solid/authorization-panel/pull/216#discussion_r665230245
 
-The problem here is that we need to make a copy of `</weekly-status/2021-04-28/.acl>` in the new acl, which means that changes to the root acl do not get propagated, and the more changes there are the more places will need to be edited to make any changes.
+As a result the acl's for all the resources could be pointing to the root acl. But doing that will tend to place all the ACLs on a Pod together, giving anyone with access to that acl, visibility on all the access control rules of the Pod.
```

Comment:
I had been thinking, and am fairly sure now, that `acl` should be made all-caps throughout.  I'm only doing so on this paragraph, but suggest that this soon be carried throughout.
```suggestion
As a result the ACLs for all the resources could be pointing to the root ACL. But doing that will tend to place all the ACLs on a Pod together, giving anyone with access to that ACL, visibility on all the access control rules of the Pod.
```